_docs/README.md

```diff
@@ -27,6 +27,8 @@ The technical specifications of the environment are as follows:
 * [PHP 7.0](http://php.net/), with the FPM SAPI and OPcache
 * [PostgreSQL 9.5](http://www.postgresql.org/)
 * [Selenium](http://www.seleniumhq.org/) hub and nodes.
+* Single sign on with [SimpleSAMLphp](https://simplesamlphp.org/)
+* Saml data store with [Redis](https://redis.io/)
```

Comment:
Can we make this more generic? "Caching with Redis" or similar? Would also be good to include the major version number.